playground/AzurePublisher/AzurePublisher.Web/wwwroot/lib/bootstrap/dist/css/bootstrap-grid.css

```diff
@@ -0,0 +1,4085 @@
+/*!
```

Comment:
bootstrap is 9MB, do you need it for this one playground app?

Reply:
Removed